requirements.txt

```diff
@@ -244,7 +244,6 @@ google-cloud-audit-log==0.2.0 \
     --hash=sha256:d0852f525ad65705f9fbff6288be4493e1449a7906fb5e01bd71c8d1e424d1fc \
     --hash=sha256:ecc7f5f2168ad4014331d6397fcea3750b2d41900f0ef6d1081cd78b3a6420e9
     # via google-cloud-logging
-google-cloud-bigquery==1.28.0 \
```

Comment:
Why is this change needed?

Reply:
It is necessary dependency updates because the CI started failing with the previous versions, today. (There's another PR open for the this dependency update, however, I included this because I needed to see if the tests are passing on CI or not. I will remove the dependency part in the next commit or will keep it, if it gets ready to merge before the other PR)
Additionally, this PR addresses a flaky test issue that was occurring across multiple PRs. This was the reason for pinning you twice. However, I will mind this for next time.